### PR TITLE
React week2

### PR DESCRIPTION
**Pull request by Rajesh-Bhatt**
I have done the following tasks for homework requirement of react-week2:

Created component AddWishlistItem.js,
Created component PlanetCard.js,
Created component SocialMediaItem.js,
Created Navbar.js component,
Footer enhanced with socialmedia icons,
Moved PlanetWishListItem.js to separate component file,
Added a link to navigate to each of the pages,
Some of styles changed in css files,
Added the functionality to select or remove of planets and so on...

### CODE REVIEW

File: app/destination/page.js [hunk 1]
```diff
@@ -1,54 +1,77 @@
...
+      description:
+        "Titan, Saturn's largest moon, is a world of dense atmosphere and liquid methane lakes. This enigmatic moon is shrouded in a thick orange haze, concealing a landscape that is both alien and strangely familiar, beckoning explorers to uncover its secrets.",
+      thumbnail: "/destination/image-titan.png",
+      isSelected: false,
+    },
+  ];
 
-  let isPlanetSelected = false;
-  let numberOfPlanets = 0;
+  const [selectedPlanets, setSelectedPlanets] = useState([]);
```
Comment (vdaguenet) [inline app/destination/page.js:38]: You could use the `planetItems` array to initialize the state

File: app/destination/page.js [hunk 2]
```diff
@@ -74,41 +98,20 @@ export const Destinations = () => {
...
-            </button>
-          </div>
+          {planetItems.map((item, index) => (
+            <PlanetCard
+              name={item.name}
+              key={index}
+              description={item.description}
+              thumbnail={item.thumbnail}
+              isSelected={selectedPlanets.includes(item.name)}
+              onAddOrRemovePlanet={() => onAddOrRemovePlanet(item.name)}
```
Comment (vdaguenet) [inline app/destination/page.js:108]:
```suggestion
              onAddOrRemovePlanet={() => onAddOrRemovePlanet(item.name, index)}
```
The onAddOrRemovePlanet has 2 arguments

File: components/destination/PlanetCard.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      />
+      <div className={styles.planetDescription}>
+        <h2>
+          {name} {isSelected ? "- SELECTED" : ""}
+        </h2>
+        <p className={styles.planetDescription}>{description}</p>
+      </div>
+      <button
+        className="roundButton"
+        onClick={() => onAddOrRemovePlanet("Pluto", 0)}
```
Comment (vdaguenet) [inline components/destination/PlanetCard.js:26]: It would be nice to have other options than Pluto

File: app/destination/page.js (hunk 2 above)
Comment (Rajesh-Bhatt) [reply app/destination/page.js:108]: Yes, Thank you for suggestions.

File: components/destination/PlanetCard.js (hunk 3 above)
Comment (Rajesh-Bhatt) [reply components/destination/PlanetCard.js:26]: Implemented!!!

**Pull request merged**